src/domain/headlessCms/client.ts

```diff
@@ -21,7 +22,7 @@ const errorLink = onError(({ graphQLErrors, networkError }) => {
 
       Sentry.captureMessage(errorMessage);
 
-      if (import.meta.env.NODE_ENV === 'development') {
+      if (AppConfig.environment === 'development') {
```

Comment:
This also works, but I think we would want it to reflect on whether the code is in bundled prod mode:
```suggestion
      if (!AppConfig.isAppInProductionMode) {
```

Reply:
The 2 main options of the old value `import.meta.env.NODE_ENV` are "development" and "production". That is what we can get with `AppConfig.isAppInProductionMode`. There could have also been NODE_ENV == "test", but that is as well something that we would have liked to have in unit testing. That's why I think using `isAppInProductionMode` is a better option.